pom.xml

```diff
@@ -139,7 +139,7 @@
     <derby.version>10.14.2.0</derby.version>
     <parquet.version>1.12.2</parquet.version>
     <orc.version>1.7.4</orc.version>
-    <jetty.version>9.4.44.v20210927</jetty.version>
+    <jetty.version>9.4.46.v20220331</jetty.version>
```

Comment:
Oh, could you update the dependency manifest?

Reply:
Oops, I thought it didn't generate changes when I ran, but it does. (Actually it caused a bunch of unrelated changes in the hadoop 2 profile, but I reverted those.) Added.

Reply:
Yes, it was a bug of Apache Maven 3.8.5. If you use the latest version (like Home-brew maven), it ignores the dependency from profiles. Recently, I'm using `build/mvnw` always to avoid that issue.